file-uploader.js

```diff
@@ -47,8 +47,11 @@ export function fileUploader(submissionBody, submissionFormat, requestHeaders, b
   };
 
   return fileUploaderUtil.validateSubmission(submissionBody, submissionFormat, baseOptions)
```

Comment:
Can you just remove the call to `fileUploaderUtil.validateSubmission` altogether, as it seems we are not using the result anywhere?

Reply:
I do not think so, we still need to validate.

But maybe someone else has a better suggestion or agrees with you.

Reply:
No, you need it to validate that the submission structure is valid.  It will throw an error if it's not valid.